### DISCUSSION TITLE
How to use it when I want to publish my custom action to Github Marketplace

### DISCUSSION THREAD

**Discussion by sztadii** [semantic-release/semantic-release#2036] [Configuration] [answered]
How to use it when I want to publish my custom action to Github Store.
Cause for publishing to the Github Marketplace we do not need NPM_TOKEN

**Comment by gr2m** [comment, marked as answer]
set `"public": false` in your `package.json` to disable publishing to npm. Then you don't need to set `NPM_TOKEN`. Compare e.g. the setup in https://github.com/gr2m/release-notifier-action

**Reply by sztadii** [reply]
I will try, thanks :)

**Reply by sztadii** [reply]
It did not throw any error, it updated the tag, but it did not push custom action to the marketplace.
<img width="704" alt="Screenshot 2021-07-18 at 22 45 35" src="https://user-images.githubusercontent.com/29600182/126081684-dc107805-04a2-4d46-80f6-9dba228e1aa3.png">

Any idea how to publish it to the marketplace automatically?

**Reply by sztadii** [reply]
Link to the repo if you would like to check https://github.com/sztadii/rc-reminder-bot

**Reply by gr2m** [reply]
> Any idea how to publish it to the marketplace automatically?

that's not possible yet, unfortunately. I've been asking for this for a long time myself. Please contact support at https://support.github.com/contact to let them know you need to bump the prirority

**Reply by sztadii** [reply]
@gr2m ohh that sad. At least I know that semantic-release will not automate it in 100% :(
